src/mc-fle2-find-range-payload.c

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2022-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <bson/bson.h>
+
+#include "mc-fle2-find-range-payload-private.h"
+#include "mongocrypt.h"
+#include "mongocrypt-buffer-private.h"
+
+void
+mc_FLE2FindRangePayload_init (mc_FLE2FindRangePayload_t *payload)
+{
+   BSON_ASSERT_PARAM (payload);
+   memset (payload, 0, sizeof (mc_FLE2FindRangePayload_t));
+   _mc_array_init (&payload->edgeFindTokenSetArray,
+                   sizeof (mc_EdgeFindTokenSet_t));
+}
+
+static void
+mc_EdgeFindTokenSet_cleanup (mc_EdgeFindTokenSet_t *etc)
+{
+   BSON_ASSERT_PARAM (etc);
```

Comment:
```suggestion
   if (!etc) {
      return;
   }
```
Suggest `if (!etc) { return; }` for consistency with other cleanup functions.